sk-csharp-console-chat/README.md

```diff
@@ -0,0 +1,87 @@
+# Semantic Kernel C# Console Chat Starter
+
+The `sk-csharp-console-chat` console application demonstrates how to use the Semantic Kernel OpenAI chat completion service.
+
+## Prerequisites
+
+- [.NET 6](https://dotnet.microsoft.com/download/dotnet/6.0) is required to run this starter.
+- Install the recommended extensions
+  - [C#](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp)
+  - [Semantic Kernel Tools](https://marketplace.visualstudio.com/items?itemName=ms-semantic-kernel.semantic-kernel)
+
+## Configuring the starter
+
+The starter can be configured by using either:
+
+- Enter secrets at the command line with [.NET Secret Manager](#using-net-secret-manager)
+- Enter secrets in [appsettings.json](#using-appsettingsjson)
+
+For Debugging the console application alone, we suggest using .NET [Secret Manager](https://learn.microsoft.com/aspnet/core/security/app-secrets) to avoid the risk of leaking secrets into the repository, branches and pull requests.
+
+### Using .NET [Secret Manager](https://learn.microsoft.com/aspnet/core/security/app-secrets)
+
+Configure an OpenAI endpoint
+
+```bash
+cd sk-csharp-console-chat
+dotnet user-secrets set "serviceType" "OpenAI"
+dotnet user-secrets set "serviceId" "gpt-3.5-turbo"
+dotnet user-secrets set "deploymentOrModelId" "gpt-3.5-turbo"
+dotnet user-secrets set "apiKey" "... your OpenAI key ..."
+```
+
+Configure an Azure OpenAI endpoint
+
+```bash
+cd sk-csharp-console-chat
+dotnet user-secrets set "serviceType" "AzureOpenAI"
+dotnet user-secrets set "serviceId" "gpt-3.5-turbo"
+dotnet user-secrets set "deploymentOrModelId" "gpt-3.5-turbo"
```

Comment:
There is a naming inconsistency between Azure OpenAI and OpenAI.

Azure uses `gpt-35-turbo` and not `gpt-3.5-turbo`